ahk/window.py

```diff
@@ -514,3 +518,51 @@ def find_windows_by_class(self, class_name, exact=False):
     def find_window_by_class(self, *args, **kwargs):
         with suppress(StopIteration):
             return next(self.find_windows_by_class(*args, **kwargs))
+
+    def run_window(self, target: str, working_dir="", options="") -> Window:
+        """Run command
```

Comment:
This is not a blocker to merging, but I would prefer if docstrings were in [sphinx docstring](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) format.

Soon, I plan to write full documentation for readthedocs in rst and, as far as i know, autodoc only uses sphinxy docstrings :)